nerfstudio/scripts/datasets/process_project_aria.py

```diff
@@ -118,25 +123,101 @@ def read_trajectory_csv_to_dict(file_iterable_csv: str) -> TimedPoses:
     )
 
 
+def undistort_image_and_calibration(
```

Comment:
maybe the changes in this file / module should be broken out into separate PR at some point.  and could probably ship sooner too then (?)